frontend/src/metabase/query_builder/components/template_tags/SnippetSidebar/SnippetSidebar.jsx

```diff
@@ -164,70 +161,46 @@ class SnippetSidebarInner extends React.Component {
                     f(this, { className: CS.mr2 }),
                   ),
                 ]}
-                {snippets.length >= MIN_SNIPPETS_FOR_SEARCH && (
-                  <Icon
-                    className={cx(S.SearchSnippetIcon, {
-                      [S.isHidden]: showSearch,
-                    })}
-                    name="search"
-                    size={HEADER_ICON_SIZE}
-                    onClick={this.showSearch}
-                  />
-                )}
+                {snippets.length >= MIN_SNIPPETS_FOR_SEARCH &&
+                  (showSearch ? (
+                    <Button variant="subtle" onClick={this.hideSearch} p="sm">
```

Comment:
- These buttons should be square, now they're 34x40
- These buttons give its container the height - let's give the container some padding instead. Buttons should not cause the container (with title) to grow